File: Br.Com.Fiap.Gere.Residuo/Model/MotoristaModel.cs
namespace Br.Com.Fiap.Gere.Residuo.Model
{
    public class MotoristaModel
    {
        public int MotoristaId { get; set; }

        // Relacionamento com AgendaModel
        public List<AgendaModel> AgendasCriadasComEsteMotorista { get; set; }

        public string MotoristaNome { get; set; }

        public string MotoristaCpf { get; set; }

        public string MotoristaNrCelular { get; set; }

        public string MotoristaNrCelularDdd { get; set; }

        public string MotoristaNrCelularDdi { get; set; }

        public bool MotoristaEstaDisponivel { get; set; } = true;
    }
}

File: BDD_Tests/BDD-Gere-Residuo/src/test/java/model/motorista/MotoristaModel.java
package model.motorista;

import com.google.gson.annotations.Expose;
import lombok.Data;

@Data
public class MotoristaModel {

    @Expose(serialize = false)
    private int motoristaId;
    @Expose(serialize = false)
    private Boolean motoristaEstaDisponivel;

    @Expose
    private String motoristaNome;

    @Expose
    private String motoristaCpf;

    @Expose
    private String motoristaNrCelular;

    @Expose
    private String motoristaNrCelularDdd;

    @Expose
    private String motoristaNrCelularDdi;
}
